examples/ui.content/src/main/content/jcr_root/content/core-components-examples/library/core-content/aemform/.content.xml

```diff
@@ -34,7 +34,7 @@
                     jcr:lastModifiedBy="admin"
                     jcr:primaryType="nt:unstructured"
                     sling:resourceType="core/wcm/components/text/v2/text"
-                    text="&lt;p>The AEM Forms Container component is a server side component written in HTL, allowing to display any AEM Forms Asset like Interactive Communication or Adaptive Form"
+                    text="&lt;p>The Adaptive Forms Container component is a server side component written in HTL, allowing to display any AEM Forms Asset like Interactive Communication or Adaptive Form"
```

Comment:
If we are changing to Adaptive Form container - it should not mention Interactive Communication